analysis_options.yaml

```
@@ -28,3 +28,6 @@ include: package:lints/recommended.yaml
 
 # For additional information about configuring this file, see
 # https://dart.dev/guides/language/analysis-options
+
+formatter:
+  trailing_commas: preserve
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)